src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,42 @@
+.stars {
+  display: flex;
+  justify-content: flex-start;
+  flex-direction: row;
+  padding-inline-start: 0;
+  list-style-type: none;
+  margin-bottom: 24px;
+  margin-right: 0;
+  margin-top: 16px;
+
+  &__star {
+    display: inline-block;
+    vertical-align: middle;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
+
+    &:last-child {
+      position: relative;
+      margin-right: 0;
+    }
+
+    &:last-child::after {
```

Comment:
The `::after` pseudo-element should not be used on the `.stars__star` element to display content that is not directly related to the stars themselves. It would be better to create a separate element for the text 'Reviews: 5'.